db/migrate/20190326145039_remove_version_date_from_ostree_branch.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveVersionDateFromOstreeBranch < ActiveRecord::Migration[5.2]
+  def change
+    remove_column :katello_ostree_branches, :version_date, :timestamp
```

Comment:
nitpick: I don't think the data type is required here since we aren't handling 'up' and 'down' direction for the migration

Reply:
We can keep it to support rollbacks? I don't mind removing it either..